orderbook/openapi.yml

```diff
@@ -769,3 +819,86 @@ components:
       required:
         - errorType
         - description
+    OrderQuoteSide:
+      description: The buy or sell side when quoting an order.
+      oneOf:
```

Comment:
Should we differentiate between sell before fees, sell after fees? I could imagine a market maker that wants to match a buy order trying to sell exactly the amount that is being bought and then add the fees on top (I agree this is less likely than the one we already have but could still be useful).

Reply:
Added. You should now be able to `POST`:
```js
{
  // bunch of order stuff...
  // Either
  kind: "sell",
  sellAmountBeforeFee: "100000",
  // Or
  kind: "sell",
  sellAmountAfterFee: "100000",
  // Or
  kind: "buy",
  buyAmountAfterFee: "100000",
}
```

The names were chosen to be consistent with the rest of the API.